pkg/cmd/roachtest/roachtestutil/mixedversion/mutators.go

```diff
@@ -652,3 +652,118 @@ func selectPartitions(
 	}
 	return option.NodeListOption{partitionedNode}, leftPartition, rightPartition
 }
+
+const (
+	// UpreplicatePanicNode is a mutator that will first upreplicate to 5X before
+	// panicking a node, allowing for a longer panic duration before recovery.
+	// This mutator is currently limited to tests that are running on 5+ nodes,
+	// so it is disabled by default.
+	UpreplicatePanicNode = "upreplicate_panic_node"
+)
+
+type replicatePanicMutator struct{}
+
+func (m replicatePanicMutator) Name() string {
+	return "upreplicate_panic_node"
+}
+func (m replicatePanicMutator) Probability() float64 {
+	return 0
+}
+
+func (m replicatePanicMutator) Generate(
```

Comment:
Should this be implemented as a different flavor of the regular `PanicNodeMutator`? That way they are mutually exclusive and you might be able to reuse some of the logic?